LICENSE

```diff
@@ -218,13 +218,9 @@ conditions of their respective licenses.
 The Apache Flink project bundles the following files under the MIT License:
 
  - jQuery v2.1.0. (http://jquery.com) - Copyright 2014 jQuery Foundation and other contributors
- - jQuery resize event v1.1 (http://benalman.com/projects/jquery-resize-plugin/) - Copyright (c) 2010 "Cowboy" Ben Alman
```

Comment:
I'm not a JavaScript expert but I can neither find parts of Ben's non-minified code [1] nor parts of the minified code [2] in our repository. But maybe we're using a different version somewhere.

[1] https://raw.githubusercontent.com/cowboy/jquery-resize/v1.1/jquery.ba-resize.js
[2] https://raw.githubusercontent.com/cowboy/jquery-resize/v1.1/jquery.ba-resize.min.js